packages/office-ui-fabric-react/src/components/ThemeGenerator/ThemeGeneratorPage.tsx

```diff
@@ -301,32 +301,31 @@ export class ThemeGeneratorPage extends React.Component<any, IThemeGeneratorPage
       ThemeGenerator.setSlot(
         themeRules[BaseSlots[BaseSlots.backgroundColor]],
         bgColor,
-        themeRules,
         bgColorIsDark,
         true,
         true);
       ThemeGenerator.setSlot(
         themeRules[BaseSlots[BaseSlots.primaryColor]],
         '#' + response.color.accentColor,
-        themeRules,
         bgColorIsDark,
         true,
         true);
       ThemeGenerator.setSlot(
         themeRules[BaseSlots[BaseSlots.foregroundColor]],
         getHexFromColor(response.color.dominantColorForeground, false),
-        themeRules,
         bgColorIsDark,
         true,
         true);
 
       themeRules[BackgroundImageUriKey] = {
         name: BackgroundImageUriKey,
-        value: 'url(\'' + this._imgUrl + '\')'
+        value: 'url(\'' + this._imgUrl + '\')',
```

Comment:
this looks...sketchy

Reply:
It doesn't look like this will be populated at any time. Do we have plans to add theme from image in the future? Should this just get taken out.

Reply:
It works, I've tested it.
Doing theme-from-image right now requires you to uncomment some code and enter your own subscription key. It works end-to-end, but needs a loooooooot of design work before it can ship. But until then, we want to make it available to those intrepid enough to find it.